packages/lodestar/src/chain/eventHandlers.ts

```diff
@@ -322,14 +322,9 @@ export async function onErrorBlock(this: BeaconChain, err: BlockError): Promise<
   // err type data may contain CachedBeaconState which is too much to log
   this.logger.error("Block error", {slot: err.signedBlock.message.slot, errCode: err.type.code});
 
-  if (err.type.code === BlockErrorCode.FUTURE_SLOT) {
```

Comment:
we need to modify this https://github.com/ChainSafe/lodestar/blob/6cf7c2c3f3df6012bd4ce773ab3b5310c347e884/packages/lodestar/src/chain/blocks/validate.ts#L38

to allow some tolerance as well, otherwise node will be out of synced for some slots

Reply:
I've added the functionality of a simple queue if the block in between 0 and 500ms early with a promise in the validation function itself. Do you think this is good enough?